### DISCUSSION TITLE
notification row background

### DISCUSSION THREAD

**Discussion by Aligehri** [ErikReider/SwayNotificationCenter#448] [Q&A]
Does anyone know why my notification row background looks gray?

![2024-06-30-174355_hyprshot](https://github.com/ErikReider/SwayNotificationCenter/assets/133458600/2ca8752e-d2a0-4d25-b55e-fa748174be0e)

I think there is nothing wrong with my code:

.notification-row {
  background: transparent;
  border-radius: 12px;
  padding: 2px;

}

.notification-row:focus,
.notification-row:hover {
  background: transparent;
}

.notification {
  border-radius: 12px;
  margin: 6px 12px;
  padding: 2px 2px 2px 2px;
  background: rgba(0,0,0,0);
}

There's no any gray color in my css.

**Comment by ffs-akl** [comment]
i don't know why it is . maybe you can try to look into  the node` .control-center .notification-row `and` .notification-group`
